packages/venia-ui/lib/targets/venia-ui-intercept.js

```diff
@@ -122,6 +122,12 @@ module.exports = targets => {
             pattern: '/checkout',
             exact: true,
             path: '../CheckoutPage'
+        },
+        {
+            name: 'Reset Password',
+            pattern: '/customer/account/createPassword',
```

Comment:
The route can change. It is the route configured in the forgot password email template in the admin panel.

I don't think we have a setting for changing it without modifying the email template manually. Should we have this in the app config?

Reply:
TBH I'm not sure how much we care about the preconfigured routes. I would also probably use hyphen instead of camelcase ie `/customer/account/create-password`.

Reply:
It is the default route. If we happen to change this and someone forgets to change the template, the whole workflow fails. That is no reason to not change since someone configuring this should know for sure. I prefer your idea as well, hyphen case is correct in this case.

Reply:
I believe until such time that there is a M2 module for Magento Core that configures the front-end to be in line with PWA Studio expectations with a click of a button the default route should work.

If there was preference for something else maybe an upward rule could be added to redirect from the default to the PWA Studio expected?

Reply:
Hey @fooman, I don't think UPWARD will be bothered in this case because the link from the email will directly hit the backend server. If we have runtime UPWARD rules, we can perform such kind of mapping. @zetlen any thoughts?

Link from the email would look something like this: 

https://staging-vdt2zeq-c5v7sxvquxwl4.eu-4.magentosite.cloud/customer/account/createPassword/?email=EMAIL_ID&token=TOKEN

Reply:
@revanth0212 I do believe you might have stumbled on a bigger discussion here...

This link:
https://staging-vdt2zeq-c5v7sxvquxwl4.eu-4.magentosite.cloud/customer/account/createPassword/?email=EMAIL_ID&token=TOKEN
combined with running the upward php connector would result in this

https://github.com/magento/magento2-upward-connector/blob/develop/Plugin/Magento/Framework/App/AreaList.php#L50

making it an upward response.

The other option I consider likely is that the back-end will be configured to send out an email like
https://venia.magento.com/customer/account/createPassword/?email=EMAIL_ID&token=TOKEN
(the base url can be changed for the complete site, whereas the /customer/account/createPassword part is in the individual email template) with a separate back-end url. In this scenario upward-js would handle it.

Reply:
I am not sure if the admin panel has an option to store/change the main site URL / public URL for instance `https://venia.magento.com/`. If so, we can check to use that.

This is how the URL is generated in the email template: 

```
<a href="{{var this.getUrl($store,'customer/account/createPassword/',[_query:[token:$customer.rp_token,email:$customer.email],_nosid:1])}}" target="_blank">{{trans "Set a New Password"}}</a>
```

Ill have to look into the implementation of the `this.getUrl` function to know what variables it is considering.

Reply:
@revanth0212 it is all configurable under Stores > Configuration > Web.

I guess the point I was trying to make is that upward is part of it one way or the other (upward-php or upward-js).